src/core/storage/sframe_interface/unity_sarray.cpp

```diff
@@ -1764,6 +1764,19 @@ std::shared_ptr<unity_sarray_base> unity_sarray::tail(size_t nrows) {
   return copy_range(start, 1, end);
 }
 
+std::shared_ptr<unity_sarray_base> unity_sarray::make_uniform_int_array(size_t size, size_t max_int) {
+  int seed_hash = 12; // XXX: how to set this properly?
+
+  auto seq = std::static_pointer_cast<unity_sarray>(
+    unity_sarray::create_sequential_sarray(size, 0, false));
+
+  auto func = [max_int, seed_hash](const flexible_type& val)->flexible_type {
+    uint64_t d = hash64(val.get<flex_int>() ^ seed_hash);
```

Comment:
Use `hash64_combine(hash64(val.get<flex_int>(), seed_hash);`

Reply:
Done.

FYI: I didn't call `hash64` on the seed. Doesn't really seem like there's much point to doing that.